examples/machine-controller.yaml

```diff
@@ -189,6 +189,75 @@ spec:
               port: 8085
             periodSeconds: 5
 ---
+apiVersion: apps/v1beta2
```

Comment:
Until I put this into a dedicated Deployment, the e2e tests always had some failures because of a connection reset when the APIServer tried to call the Webhook